PubSub/src/Topic.php

```diff
@@ -50,6 +50,10 @@ class Topic
     use ArrayTrait;
     use ResourceNameTrait;
 
+    const DEFAULT_COMPRESSION_BYTES_THRESHOLD = 240;
+
+    const DEFAULT_ENABLE_COMPRESSION = false;
```

Comment:
We should make these `private const`

Reply:
They are used in tests too thus cannot make them private. Do we have any other option?